utils/parser-lib/profile-input.swift

```diff
@@ -294,7 +294,7 @@ class BrowserViewController: UIViewController {
             self.displayedPopoverController = nil
         }
 
-        // If we are displying a private tab, hide any elements in the tab that we wouldn't want shown
+        // If we are displaying a private tab, hide any elements in the tab that we wouldn't want shown
```

Comment:
I would prefer to keep this file as-is because this is a real-world Swift file that we use to make profile-guided optimizations for the SwiftSyntax parser. For that I would prefer to keep it in its original state. 

In practice it shouldn’t really matter because the changes are unlikely to affect parsing in any significant way, but still.